src/OSSupport/Network.h

```diff
@@ -0,0 +1,215 @@
+
+// Network.h
+
+// Declares the classes used for the Network API
+
+
+
+
+
+#pragma once
+
+
+
+
+
+/** Interface that provides the methods available on a single TCP connection. */
+class cTCPLink
```

Comment:
Also is cTCPLick intended to be used as a value type as it only contains a pointer?